### PR TITLE
fix: Update jimmidyson/configmap-reload image to fix CVE

### PR DESCRIPTION
**Pull request by jimmidyson**
**What type of PR is this?**
Bug

**What this PR does/ why we need it**:
fix: Update jimmidyson/configmap-reload image to fix [CVE](https://nvd.nist.gov/vuln/detail/CVE-2018-1000500). [Upstream PR](https://github.com/prometheus-operator/kube-prometheus/pull/894) also submitted.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by armandgrillet — APPROVED** [review]
LGTM

**Review by makkes — APPROVED** [review]

File: staging/prometheus-operator/values.yaml [hunk 1]
```diff
@@ -1439,7 +1439,7 @@ prometheusOperator:
   ##
   configmapReloadImage:
     repository: docker.io/jimmidyson/configmap-reload
```
Comment (hectorj2f) [inline staging/prometheus-operator/values.yaml:1441]: Would it make sense to have this repo under our org in a fork ?

Comment (jimmidyson) [reply staging/prometheus-operator/values.yaml:1441]: This is a pretty popular public image, it's used by lots of places (including upstream prometheus-operator where this came from, and funnily enough even my bank use it...).

Comment (gracedo) [reply staging/prometheus-operator/values.yaml:1441]: Can we make sure to contribute this upstream? Since this is a fork, so I'm always weary of making chart changes - this can also go in kba otherwise, but since it's a cve priority maybe let's just make this change here but also open up an upstream PR. There is also a `patch` dir here that we've also added with scripts that make mesosphere-specific chart changes, to make upgrades easier, but I'm okay skipping that if this will go into upstream anyways.

tl;dr let's open up an upstream pr!

Comment (hectorj2f) [reply staging/prometheus-operator/values.yaml:1441]: 😆 , it is all about popularity, 👍🏻 .

**Review by gracedo — APPROVED** [review]
🙈 my bad, I am guilty of not reading the PR description!! upstream pr submitted thanks 👍

**Comment by gracedo** [comment]
and it looks like upstream chart [removed jimmi's reloader img](https://github.com/prometheus-community/helm-charts/pull/582/files#diff-c0fdbc5c26d2f602485f168b5a55814cd73bd3347907c5097395120d64c2f445L1438-L1444) (sorry! :( ) with the upgrade to prometheus-operator 0.45.0 so this change won't need to persist across the next time we upgrade our prom fork

File: staging/prometheus-operator/values.yaml (hunk 1 above)
Comment (jimmidyson) [reply staging/prometheus-operator/values.yaml:1441]: @gracedo I already pushed it upstream and it's been merged - https://github.com/prometheus-operator/kube-prometheus/pull/894
